pyproject.toml

```diff
@@ -55,11 +55,12 @@ sphinxcontrib-openapi = {version="^0.8.3", optional=true}
 mypy = {version="1.10.1", optional=true}
 requests = {version = "^2.32.3", optional = true}
 healpy = {version = "^1.18.0", optional = true}
+netcdf4 = {version ="^1.7.2",  optional = true}
```

Comment:
I can't comment on the line above, but we need to bump the xarray minimum dependency as well to [v2024.10.0](https://github.com/pydata/xarray/releases/tag/v2024.10.0)

That is the first version with datatree support.
https://xarray.dev/blog/datatree